graphql-java-kickstart/src/main/java/graphql/kickstart/execution/subscriptions/apollo/SubscriptionConnectionInitCommand.java

```diff
@@ -19,6 +19,7 @@ public void apply(SubscriptionSession session, OperationMessage message) {
       connectionListeners.forEach(it -> it.onConnect(session, message));
       session.sendMessage(new OperationMessage(Type.GQL_CONNECTION_ACK, message.getId(), null));
     } catch (Throwable t) {
+      log.error("Cannot initialize subscription command '{}': {}", message, t);
```

Comment:
Why did you add the `: {}` part here? Since the way it was worked just fine letting it print the stack trace.